code/modules/clothing/factions/suns.dm

```diff
@@ -0,0 +1,572 @@
+	name = "eye mask health scanner HUD"
+	desc = "A peculiar looking mask commonly seen at academic functions. This one has a health HUD lense in it."
+	icon_state = "suns_doctormask"
+	item_state = "suns_doctormask"
+	glass_colour_type = /datum/client_colour/glass_colour/lightblue
+	icon = 'icons/obj/clothing/faction/suns/eyes.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/eyes.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/glasses/hud/security/suns
+	name = "eye mask security HUD"
+	desc = "A peculiar looking mask commonly seen at academic functions. This one gives a heads-up display that scans the humanoids in view and provides accurate data about their ID status and security records."
+	icon_state = "suns_pkmask"
+	item_state = "suns_pkmask"
+	glass_colour_type = /datum/client_colour/glass_colour/red
+	icon = 'icons/obj/clothing/faction/suns/eyes.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/eyes.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+
+//////////
+//Masks//
+/////////
+
+
+/obj/item/clothing/mask/gas/suns //someone mentioned they were interested in using this sprite as the regular gasmask re-sprite, I forgor who so we'll deal with that when I PR this
+	name = "black gas mask"
+	desc = "A black face covering that allows the user to connect to a personal gas supply. Suprisingly not great at preventing gas inhalation."
+	icon_state = "suns_gasmask"
+	item_state = "suns_gasmask2"
+	icon = 'icons/obj/clothing/faction/suns/mask.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/mask.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/mask/surgical/suns
+	name = "purple sterile mask"
+	desc = "A sterile mask designed to help prevent the spread of diseases. Now in purple! Pretty!"
+	icon_state = "suns_sterile"
+	item_state = "suns_doctorcap"
+	icon = 'icons/obj/clothing/faction/suns/mask.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/mask.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+/obj/item/clothing/mask/breath/suns
+	name = "\improper SUNS half face mask"
+	desc = "A close-fitting mask that covers JUST enough to connect an air supply."
+	icon_state = "suns_captainmask"
+	item_state = "suns_captainmask"
+	icon = 'icons/obj/clothing/faction/suns/mask.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/suns/mask.dmi'
+	lefthand_file = 'icons/mob/inhands/faction/suns/suns_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/faction/suns/suns_righthand.dmi'
+
+
+///////////
+//Gloves//
+//////////
+
+
+/obj/item/clothing/gloves/color/captain/suns
```

Comment:
For gloves it'd also be better `.../clothing/gloves/suns/etc...`
Just take the siemens coeffecient var from insulated gloves and put it on ones that need it. `/color` subtype not necessary. I think having that would let them be dyed in washing machines anyways (undesired behavior)